### ISSUE TITLE
Pistol bug

### ISSUE THREAD

**Issue by hpksoldati** [dionpirotta/Custom-CSGO-HUD#8]
When the player drops the pistol without having a primary weapon. It shows that an image is missing.

![image](https://user-images.githubusercontent.com/16945944/79288349-dadaed00-7e9c-11ea-91b7-25a4c10de313.png)

Good Job guys !! ty

**Comment by dionpirotta** [comment]
This would be due to apex having a knife (one of the new ones) that I dont have the image for

**Comment by hpksoldati** [comment]
I solved the problem, how can i send you the images? I think there are missing 4-5 new knifes.

Can i send you thru discord? Thanks again!

**Comment by dionpirotta** [comment]
Yeah sure thing, been working on the new hud system so havent touched this one in ages.

Discord is Komodo#3705
________________________________
